src/Playwright/Playwright.csproj

```diff
@@ -18,7 +18,7 @@
     <ReleaseVersion>0.0.0</ReleaseVersion>
     <BuildFromSource>True</BuildFromSource>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
-    <NoWarn>1701;1702;CS0067;1734</NoWarn>
+    <NoWarn>1701;1702;CS0067;1734;NU5110;NU5111</NoWarn>
```

Comment:
These warnings are for PowerShell files O.o did you add `ps1` files, or is this a weird side-effect of something?

Reply:
I have no idea, Dario told me to add those.